packages/web-react/src/components/Toast/README.md

```diff
@@ -353,9 +353,11 @@ const { show } = useToast();
          │                 ┌─⫸ ToastBar ID (required)
          │                 │
 show('Toast message', 'toast-id', {
+  autoCloseInterval: 3000  // Set interval after ToastBar will be closed, default: 3000
```

Comment:
Please, add units into the comments, e.g. miliseconds.

Reply:
units added